### ISSUE TITLE
Signup: `delaySearch` propTypes warning on domain step

### ISSUE THREAD

**Issue by sirbrillig** [Automattic/wp-calypso#2772]
When visiting the "Let's find a domain" step of the signup flow in a development environment, this error appears in the console:

```
Warning: Failed propType: Invalid prop `delayTimeout` of type `string` supplied to `Search`, expected `number`. Check the render method of `SearchCard`.
```

I believe the error is coming from [this commit](https://github.com/Automattic/wp-calypso/commit/833f9209795fe8c3fa2feceeaff2367bfdfb84bb) which passes the number `2000` as a String, rather than an number (ping @dzver).

I think it may work to pass the number escaped from JSX like this `{2000}`, or we should modify the `Search` component to accept strings and convert them to integers.

**Comment by dzver** [comment]
It's in progress https://github.com/Automattic/wp-calypso/pull/2689

**Comment by fabianapsimoes** [comment]
Fixed in #2689.
